### DISCUSSION TITLE
Azuracast Broadcasting Power Info

### DISCUSSION THREAD

**Discussion by xfiniteradiomedia** [AzuraCast/AzuraCast#6921] [Show and tell]
I just wanted to post this for those wanting to know the stats of streaming channels as far as the amount of cpu and processing you need. I am currently using a 32gb 1tb 12cpu AMD 3200 tower and I'm currently able to broadcast 75 streaming channels at once and using 30 percent of my procesing power. I have another old tower with 4gb 250gb Amd processor and it can only run 7 channels before it starts locking up. Just thought I'd share it so you can have an idea of what you need.

**Comment by eranwo** [comment]
@xfiniteradiomedia Thanks for sharing.

Are you experiencing sync issues in "Now Playing"?
I mean that the name of the song in the WEB interface and in the API appears about 30 seconds or more after the song changes.

Not like on Spotify where the "Now Playing" is displayed exactly with the songs. Updated in perfect synchronization
